server/etcdmain/help.go

```diff
@@ -277,9 +277,11 @@ Experimental feature:
     Enable to check data corruption before serving any client/peer traffic.
   --experimental-corrupt-check-time '0s'
     Duration of time between cluster corruption check passes.
-  --experimental-compact-hash-check-enabled 'false'
+  --experimental-compact-hash-check-enabled 'false'. It's deprecated, and will be decommissioned in v3.7. Use '--feature-gates=CompactHashCheck=true' instead.
     Enable leader to periodically check followers compaction hashes.
   --experimental-compact-hash-check-time '1m'
+    Duration of time between leader checks followers compaction hashes. It's deprecated, and will be decommissioned in v3.7. Use '--compact-hash-check-time' instead.
+  --compact-hash-check-time '1m'
```

Comment:
Please update the description per https://github.com/etcd-io/etcd/pull/19053#discussion_r1883688023